### PR TITLE
Add extract_units_drift_vector

### PR DESCRIPTION
**Pull request by samuelgarcia**
This will be usefull for benchmark estimate_motion().
This give the summed of all units drift vector.

@yger

### CODE REVIEW

File: MEArec/tools.py [hunk 1]
```diff
@@ -2498,6 +2498,78 @@ def compute_drift_idxs_from_drift_list(spike_index, spike_train_frames, drift_li
...
+
+    if mearec_file is not None:
+        recgen = load_recordings(mearec_file)
+
+    drift_list = recgen.drift_list
+    locations = np.array(recgen.template_locations)
+
+    if time_vector is None:
+        # the main times constructed from the first drift
+        drift_dict = drift_list[0]
```
Comment (alejoe91) [inline MEArec/tools.py:2538]: maybe argmax of `fs` is better

File: MEArec/tools.py [hunk 2]
```diff
@@ -2498,6 +2498,78 @@ def compute_drift_idxs_from_drift_list(spike_index, spike_train_frames, drift_li
...
+            interpolated_drift_vector_idxs = drift_vector_idxs
+        else:
+            # linear interpolation on the timevector
+            local_times = np.arange(drift_vector_idxs.shape[0]) / drift_fs
+            f = scipy.interpolate.interp1d(local_times, drift_vector_idxs)
+            interpolated_drift_vector_idxs = f(time_vector)
+        drift_dict['interpolated_drift_vector_idxs'] = interpolated_drift_vector_idxs
+
+    n_units = len(recgen.spiketrains)
+    drift_vectors = np.zeros((time_vector.size, n_units), dtype='float32')
```
Comment (alejoe91) [inline MEArec/tools.py:2560]:
```suggestion
    unit_drift_vectors = np.zeros((time_vector.size, n_units), dtype='float32')
```

File: MEArec/tools.py (hunk 1 above)
Comment (samuelgarcia) [reply MEArec/tools.py:2538]: done

File: MEArec/tools.py (hunk 2 above)
Comment (samuelgarcia) [reply MEArec/tools.py:2560]: done

**Comment by samuelgarcia** [comment]
@alejoe91 : I think this is ready

**Pull request merged**